From ab6db30b17a5135b6934f3ccaefe04d3cbc3d838 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Wed, 20 Sep 2023 16:04:08 +0000
Subject: IWYU: add math.h for sqrt in base::MovingWindow

Bug: 957519
Change-Id: I6a6f4821fea5604ea7b8ff278fa1e641b4863951
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4877015
Reviewed-by: Daniel Cheng <dcheng@chromium.org>
Commit-Queue: Daniel Cheng <dcheng@chromium.org>
Auto-Submit: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#1199039}
---
 base/moving_window.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base/moving_window.h b/base/moving_window.h
index c50da69903af2..1b5c8768415e4 100644
--- a/base/moving_window.h
+++ b/base/moving_window.h
@@ -5,6 +5,7 @@
 #ifndef BASE_MOVING_WINDOW_H_
 #define BASE_MOVING_WINDOW_H_
 
+#include <math.h>
 #include <stddef.h>
 
 #include <functional>
-- 
2.39.2

